lib/src/main/java/org/exoplatform/social/addons/storage/entity/IdentityEntity.java

```diff
@@ -77,6 +80,12 @@
   @OneToOne(mappedBy = "identity", fetch = FetchType.LAZY, orphanRemoval = true)
   private ProfileEntity profile;
 
+  @OneToMany(mappedBy = "receiver", fetch = FetchType.LAZY, cascade = {CascadeType.REMOVE}, orphanRemoval = true)
+  private List<Connection> incomingConnections;
+
+  @OneToMany(mappedBy = "sender", fetch = FetchType.LAZY, cascade = {CascadeType.REMOVE}, orphanRemoval = true)
+  private List<Connection> outgoingConnections;
+
   public long getId() {
     return id;
   }
```

Comment:
To be compatible with usage of core Social API, we could provide #getStringId() method to return ID in string. It helps to simplify the code of checking condition in many other places.